src/pages/ReportByName.js

```diff
@@ -1,29 +1,21 @@
-import React, { useRef,useEffect } from "react";
-import { useState } from "react";
+import React, { useEffect, useState } from "react";
 import SideBarComponent from "../components/SidebarComponent";
-import AttendanceService from "../Services/AttendanceService";
 import ReportService from "../Services/ReportService";
-import UserService from "../Services/UserService";
-import "./Report.css"
+import "./Report.css";
 
 const Reportbyname = () => {
```

Comment:
Always use camel casing for component names

Reply:
updated component names